nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -4,10 +4,28 @@
 # the user/group to run under.)
 
 { config, lib, ... }:
+let
+  global-config = config;
+in
+{ config, name, ... }:
 
 with lib;
 {
   options = {
+    assertions = mkOption {
+      type = types.listOf (types.attrsOf types.anything);
```

Comment:
fwiw the assertion module does `types.unspecified` here instead of `anything`? Any reason this isn't done here?

Reply:
Not that I can remember